robot_scripts/PageObjects/dashboard.robot

```diff
@@ -48,8 +48,8 @@ Click Merits Link When Visible
 Click Merits Case Search Link When Visible
     [Documentation]  Uses: Selenium. Retruns: none.
     ...   Click on the cases and clients link.
-    Wait Until Element Is Visible    css:li[id='CCMS Complex Merits Caseworker']
-
+    #Wait Until Element Is Visible    css:li[id='CCMS Complex Merits Caseworker']
```

Comment:
can we remove the commented-out lines please?

Reply:
Yes, shall I do it or would you?

Reply:
Done now.

Reply:
👍